spec/integration/aggregate_spec.rb

```diff
@@ -29,7 +29,7 @@ def populate_logs(job_count: 10, parts_count: 50)
     lps = Travis::Logs::LogPartsWriter.new
 
     job_count.times do |n|
-      job_id = 17_321 + n
+      job_id = Random.rand(20_000) + n
```

Comment:
I'm not sure how tests work (I couldn't really figure out where the actual log records are created, because `LogPartsWriter` seem to create only log parts), so I'm also not really sure why this works, but setting it to random value seems to fix the issue of 2 tests affecting each other (previously there was only one test in this file, so there were no problems with it).